### PR TITLE
Make title placement consistent

### PR DESCRIPTION
**Pull request by BarryPiccinni**
Move the title property in each schema to be in a consistent place.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1906,7 +1906,7 @@ module.exports = {
                 },
                 'q-applicant-building-and-street-2': {
                     type: 'string',
-                    title: "<span class='govuk-visually-hidden'>Building and street line 2</span>",
```
Comment (armoj) [inline index.js:1909]: I remember the chat around this change. But will this temporarily be visible to the user, or have we done something to hide it by other means at this point?

File: index.js [hunk 2]
```diff
@@ -449,11 +449,11 @@ module.exports = {
         'p--whats-the-crime-reference-number': {
             $schema: 'http://json-schema.org/draft-07/schema#',
             type: 'object',
-            title: "What's the crime reference number?",
```
Comment (armoj) [inline index.js:452]: can we single-quote this (and any others to make that consistent). I'm surprised the linter didn't catch these.

File: index.js [hunk 3]
```diff
@@ -1448,11 +1448,11 @@ module.exports = {
         'p-offender-do-you-know-the-name-of-the-offender': {
             $schema: 'http://json-schema.org/draft-07/schema#',
             type: 'object',
-            title: "Do you know the offender's name?",
```
Comment (armoj) [inline index.js:1451]: single-quote

File: index.js [hunk 4]
```diff
@@ -1483,11 +1483,11 @@ module.exports = {
         'p-offender-enter-offenders-name': {
             $schema: 'http://json-schema.org/draft-07/schema#',
             type: 'object',
-            title: "Enter the offender's name",
```
Comment (armoj) [inline index.js:1486]: single-quote

File: index.js [hunk 5]
```diff
@@ -2801,10 +2801,10 @@ module.exports = {
         'p-applicant-when-will-you-find-out': {
             $schema: 'http://json-schema.org/draft-07/schema#',
             type: 'object',
-            title: "When will you find out if you've been awarded compensation?",
```
Comment (armoj) [inline index.js:2804]: single-quote

**Review by armoj — CHANGES_REQUESTED** [review]
Looks good. Just a couple of smalls things and 1 question

File: index.js (hunk 1 above)
Comment (BarryPiccinni) [reply index.js]: This has been reverted.

**Review by armoj — APPROVED** [review]

**Pull request merged**